funding/manager.go

```diff
@@ -2418,18 +2418,11 @@ func (f *Manager) fundingTimeout(c *channeldb.OpenChannel,
 	go func() {
 		defer f.wg.Done()
 
-		peerChan := make(chan lnpeer.Peer, 1)
-		var peerKey [33]byte
-		copy(peerKey[:], c.IdentityPub.SerializeCompressed())
-
-		f.cfg.NotifyWhenOnline(peerKey, peerChan)
-
-		var peer lnpeer.Peer
-		select {
-		case peer = <-peerChan:
-		case <-f.quit:
+		peer, err := f.waitForPeerOnline(c.IdentityPub)
+		if err != nil {
```

Comment:
nit: perhaps add a comment in this if-statement saying that the only error that could possibly be returned from the func would be if the peer is shutting down. Just to justify why we can return from the go-routine on getting an error here